ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -264,6 +264,37 @@ conformance_test(
     ],
 )
 
+conformance_test(
```

Comment:
FYI @mziolekda @meiersi-da 
Re verifying that TLS 1 and 1.1 are disallowed (https://github.com/digital-asset/daml/pull/10898#pullrequestreview-756348688) this a conformance test for that.